gql/createPost.gql

```diff
@@ -0,0 +1,9 @@
+mutation createPost($title: String!, $url: String!) {
```

Comment:
Can you revert back graphQL files? I mean you can copy-paste them to each component. So basically we will use them .gql files as you created. Not like the old way.